packages/payments-plugin/e2e/mollie-payment.e2e-spec.ts

```diff
@@ -324,6 +335,31 @@ describe('Mollie payments (with useDynamicRedirectUrl set to true)', () => {
             });
         });
 
+        it('Should reuse payment url when amount is the same', async () => {
```

Comment:
The test if a new order is created when amounts are **not** the same, is already being done implicitly in the next test `Should get payment url with deducted amount if a payment is already made`